docs/application/dotnet/tutorials/sec-privileges.md

```diff
@@ -140,24 +143,27 @@ when using security-sensitive API modules in .net applications:
 | `http://tizen.org/privilege/recorder` | public | Microphone | The application can record video and audio. |
 | `http://tizen.org/privilege/screenshot` | platform |  | The application can capture screenshots. |
 | `http://tizen.org/privilege/secureelement` | public |  | The application can access secure smart card chips such as UICC/SIM, embedded secure elements, and secure SD cards. |
+| `http://tizen.org/privilege/securesysteminfo` | partner | Device unique ID | The application can read non-resettable secure device information. |
 | `http://tizen.org/privilege/shortcut` | public |  | The application can create and delete shortcuts. |
 | `http://tizen.org/privilege/softap` | public |  | The application can manage SoftAP configuration settings, such as the Service Set Identifier (SSID) and password. |
 | `http://tizen.org/privilege/softap.admin` | platform |  | The application can turn SoftAP on or off, and change its settings. |
 | `http://tizen.org/privilege/systemmonitor` | public |  | The application can read system information, including information from the CPU and RAM. |
 | `http://tizen.org/privilege/systemsettings` | public |  | The application can read and write unrestricted system settings. Deprecated since 2.3.1. |
 | `http://tizen.org/privilege/systemsettings.admin` | platform |  | The application can read and write all system settings. |
 | `http://tizen.org/privilege/tee.client` | partner |  | The application can call security related functions running inside a Trusted Execution Environment (TEE), which ensures that sensitive data is stored, processed, and protected in an isolated, trusted environment. |
-| `http://tizen.org/privilege/telephony` | public |  | The application can retrieve telephony information, such as the network and SIM card used, the IMEI, and the statuses of calls. |
+| `http://tizen.org/privilege/telephony` | public |  | The application can retrieve telephony information, such as the network and SIM card used and the statuses of calls. |
 | `http://tizen.org/privilege/telephony.admin` | platform |  | The application can manage telephony settings, such as those for incoming and outgoing calls, forwarding and holding calls, networks, and SIM cards. |
 | `http://tizen.org/privilege/tethering.admin` | platform |  | The application can enable and disable tethering services. This may result in additional charges depending on user's payment plan. |
 | `http://tizen.org/privilege/use_ir` | public |  | The application can use the infrared transmitter. |
 | `http://tizen.org/privilege/voicecontrol.manager` | platform |  | The application can record user's voice and recognize it so that voice commands can be used to control this app. It can also give responses to user's commands. |
+| `http://tizen.org/privilege/voicecontrol.tts` | partner |  | The application can use the voice control engine to recognize users' voice commands and provide voice feedback using its own voice. |
```

Comment:
Suggestion, please change "users" to "you"

The application can use the voice control engine to recognize your voice commands and provide voice feedback using its own voice.

Reply:
Done.